lxd/api_project.go

```diff
@@ -562,6 +562,14 @@ func projectValidateConfig(config map[string]string) error {
 			continue
 		}
 
+		// Restriction keys can be set or changed only if the project
+		// has "restrict" set to true.
+		if shared.StringInSlice(key, projecthelpers.AllRestrictions) {
```

Comment:
Hmm, do we really care about this?

It won't be effective, but we have other places in our configs where you can set children key that won't be effective unless you enable the parent.

This kind of restriction can sometimes make it hard on folks using PATCH to modify their configs, so I'm not sure that we really need this check.

Reply:
Dropped.